app/views/template_inputs/_form.html.erb

```diff
@@ -4,6 +4,8 @@
       <%= text_f f, :name, :disabled => @template.locked? %>
       <%= checkbox_f f, :required, :disabled => @template.locked? %>
       <%= selectable_f f, :input_type, template_input_types_options(@template.acceptable_template_input_types), {}, :class => 'input_type_selector without_select2', :disabled => @template.locked? %>
+      <%= selectable_f f, :value_type, [[_('Plain'), 'plain'], [_("Search"), 'search'], [_('Date'), 'date']], {}, {'data-item': f.object.id, class: 'select-value-type', onchange: "tfm.templateInputs.inputValueOnchange(this)"}  %> 
+      <%= selectable_f f, :resource_type, Permission.resources_with_translations, {include_blank: _('None')}, {wrapper_class: "#{hide_class(f.object)} form-group resource-type"} %>
```

Comment:
this shouldn't include blank, or we should have a validation in model that enforces presence in case value_type is set to Search